tasks/routes/usersWithValidation.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const uuid = require('uuid');
+const Joi = require('@hapi/joi');
+const validator = require('express-joi-validation').createValidator({ allowUnknown: false });
+const router = express.Router();
+
+const userStorage = new Map();
+
+const userGetDeleteSchema = Joi.object({
```

Comment:
Consider keeping the validation next to the related route file.
For the exact file structure, refer to the previous PR comments.